app/templates/components/high-charts.hbs

```diff
@@ -1 +1,2 @@
-{{yield}}
+<div class="chart-container"></div>
```

Comment:
I would expect the container div to wrap the `yield` like so:

```handlebars
<div class="chart-container">
  {{yield chart}}
</div>
```

Can you update this? Or is there a reason why it's not wrapped?